dts/x86/yaml/arduino101.yaml

```diff
@@ -0,0 +1,42 @@
+---
+title: ardunio101 Uart
+id: intel,ardunio-uart
+version: 0.1
+
+description: >
+    This binding gives a base representation of the K64 UART
+
+inherits:
+  - !include uart.yaml
+
+properties:
+  - compatible:
+      type: string
+      category: required
+      description: compatible strings
+      constraint: "intel,ardunio101-uart"
```

Comment:
Same thing here.  I assume this was supposed to be  arduino.  At least these were all consistent